src/osmviz/manager.py

```diff
@@ -257,8 +293,11 @@ def getTileURL(self, tile_coord, zoom):
         Given x, y coord of the tile to download, and the zoom level,
         returns the URL from which to download the image.
         """
-        params = (zoom, tile_coord[0], tile_coord[1])
-        return self.server+"/%d/%d/%d.png" % params
+        return self.url \
+            .replace("{z}", str(zoom)) \
+            .replace("{x}", str(tile_coord[0])) \
+            .replace("{y}", str(tile_coord[1])) \
+            .replace("{s}", str(self.scale))
```

Comment:
Got it. I modified the `replace` call for the md5 hash computation as well, which failed because I was only replacing the "{s}" placeholder for obvious reasons. After putting back `replace` there it now runs smoothly.